src/components/pages/landing/Gooddollar.js

```diff
@@ -1,38 +1,24 @@
 import React, { useState, useEffect, useCallback } from 'react';
-import {
-  createLoginLink,
-  parseLoginResponse,
-  LoginButton,
-  useLogin,
-} from '../../../../packages/gooddollar-login-sdk';
 import { useFormik } from 'formik';
 import { toast } from 'react-toastify';
 import * as Yup from 'yup';
-import 'react-phone-number-input/style.css';
 import { CircleSpinner } from 'react-spinners-kit';
-import { wallet } from '../../..';
 import 'react-phone-number-input/style.css';
+import 'react-phone-number-input/style.css';
+import { wallet } from '../../..';
 import { verifyUser } from '../../../services/api';
 import { useUniqueGUser } from '../../../utils/uniqueUser';
 import { supabase } from '../../../utils/supabase';
 import { log_event } from '../../../utils/utilityFunctions';
 import { getConfig } from '../../../utils/config';
 
 export const Gooddollar = ({ setShowGooddollarVerification }) => {
-  const gooddollarLink = createLoginLink({
-    v: 'I-AM-HUMAN-DAPP',
-    web: 'https://i-am-human.dapp/',
-    id: '0x09D2011Ca5781CA70810F6d82837648132762F9a',
-    r: ['name'],
-    rdu: window.location.href,
-  });
-  const [rawGoodDollarData, setRawGoodDollarData] = React.useState(null);
-  const [editableFields, setEditableFields] = React.useState({
-    name: true,
-    gDollarAccount: true,
-    status: true,
+  const [editableFields, setEditableFields] = useState({
+    code: '',
+    claimer: wallet.accountId,
+    redirect_uri: '',
   });
-  const [submitting, setSubmitting] = React.useState(null);
+  const [submit, setSubmit] = useState(null);
```

Comment:
I remember you changed it someplace else, do we need to rebase or was the PR closed for that?

Reply:
because of we are starting implementing new UI\UX and I'll refactor code in some files